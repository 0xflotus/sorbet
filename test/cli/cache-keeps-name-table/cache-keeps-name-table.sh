#!/bin/bash

CACHE=$(mktemp -d)

main/ruby-typer -p name-table --cache-dir="$CACHE" test/cli/cache-keeps-name-table/cache-keeps-name-table.rb 2>&1
main/ruby-typer -p name-table --cache-dir="$CACHE" test/cli/cache-keeps-name-table/cache-keeps-name-table.rb 2>&1
