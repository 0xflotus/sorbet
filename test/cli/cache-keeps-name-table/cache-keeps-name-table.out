class <constant:<root>> < ::Object () -> <Class:<root>> @ -e:1
  class <constant:CCCCCCCCC> < ::Object () -> <Class:CCCCCCCCC> @ test/cli/cache-keeps-name-table/cache-keeps-name-table.rb:1
  class <singleton class:<constant:CCCCCCCCC>> < ::<Class:Object> () @ test/cli/cache-keeps-name-table/cache-keeps-name-table.rb:1

class <constant:<root>> < ::Object () -> <Class:<root>> @ -e:1
  class <constant:CCCCCCCCC> < ::Object () -> <Class:CCCCCCCCC> @ test/cli/cache-keeps-name-table/cache-keeps-name-table.rb:1
  class <singleton class:<constant:CCCCCCCCC>> < ::<Class:Object> () @ test/cli/cache-keeps-name-table/cache-keeps-name-table.rb:1

